external-crates/move/crates/move-compiler/tests/move_2024/expansion/meaningless_copy_move.exp

```diff
@@ -0,0 +1,136 @@
+   │         │     
+   │         Invalid 'copy' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:14:9
+   │
+14 │         copy &x.y;
+   │         ^^^^ ---- Expected a name or path access, e.g. 'x' or 'e.f'
+   │         │     
+   │         Invalid 'copy' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:15:9
+   │
+15 │         copy *&x.y;
+   │         ^^^^ ----- Expected a name or path access, e.g. 'x' or 'e.f'
+   │         │     
+   │         Invalid 'copy' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:16:9
+   │
+16 │         copy x.id();
+   │         ^^^^ ------ Expected a name or path access, e.g. 'x' or 'e.f'
+   │         │     
+   │         Invalid 'copy' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:17:9
+   │
+17 │         copy !0;
+   │         ^^^^ -- Expected a name or path access, e.g. 'x' or 'e.f'
+   │         │     
+   │         Invalid 'copy' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:18:9
+   │
+18 │         copy 0;
+   │         ^^^^ - Expected a name or path access, e.g. 'x' or 'e.f'
+   │         │     
+   │         Invalid 'copy' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:19:9
+   │
+19 │         copy 1 + 1;
+   │         ^^^^ - Expected a name or path access, e.g. 'x' or 'e.f'
+   │         │     
+   │         Invalid 'copy' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:21:9
+   │
+21 │         move &x;
+   │         ^^^^ -- Expected a name or path access, e.g. 'x' or 'e.f'
+   │         │     
+   │         Invalid 'move' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:22:9
+   │
+22 │         move *&x;
+   │         ^^^^ --- Expected a name or path access, e.g. 'x' or 'e.f'
+   │         │     
+   │         Invalid 'move' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:23:9
+   │
+23 │         move &x.y;
+   │         ^^^^ ---- Expected a name or path access, e.g. 'x' or 'e.f'
+   │         │     
+   │         Invalid 'move' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:24:9
+   │
+24 │         move *&x.y;
+   │         ^^^^ ----- Expected a name or path access, e.g. 'x' or 'e.f'
+   │         │     
+   │         Invalid 'move' of expression
+
+error[E01013]: invalid 'move' or 'copy'
+   ┌─ tests/move_2024/expansion/meaningless_copy_move.move:25:9
+   │
+25 │         move x.id();
```

Comment:
I'm surprised this one is invalid.

Reply:
What would you want it to do? Currently I'm only moving locations, not values.